larq_compute_engine/mlir/tf_to_tfl_flatbuffer.cc

```diff
@@ -82,10 +82,13 @@ Status ConvertTFExecutorToFlatbuffer(mlir::ModuleOp module, bool export_to_mlir,
   }
 
   // This is the only modification compared to the upstream tensorflow file
+  // TODO: This is no longer the case, these files have diverged since TF2.6
```

Comment:
>This is the only modification compared to the upstream tensorflow file

This is no longer the case, there is quite some diversion. Should we try to synchronize again?

Reply:
I think for now this seems to work OK. I'm happy to synchronize this in a follow up PR since it seems they added quite a few new features, but I am a bit unsure if we already need to pull in all the new experimental flags.